src/tarkov-data-manager/index.js

```diff
@@ -471,14 +463,28 @@ app.get('/items', async (req, res) => {
                                 <a class="waves-effect waves-light btn filter-types-all"><i class="material-icons left">all_inclusive</i>All</a>
                                 <a class="waves-effect waves-light btn filter-types-none"><i class="material-icons left">not_interested</i>None</a>
                             </div>
-                            <div class="switch">
+                            <div>
+                                <label>
+                                    <input class="filter-types-require-selected" name="type-filter-function" type="radio" value="any" checked>
+                                    <span>Require any</span>
+                                </label>
+                                <label>
+                                    <input class="filter-types-require-selected" name="type-filter-function" type="radio" value="all">
+                                    <span>Require all</span>
+                                </label>
+                                <label>
+                                    <input class="filter-types-require-selected" name="type-filter-function" type="radio" value="none">
+                                    <span>Exclude</span>
+                                </label>
+                            </div>
+                            <!--iv class="switch">
```

Comment:
I'm unaware of this syntax, what does it do?

Reply:
`<!--` and `-->` are the opening and closing tags of HTML comments. I replaced a switch with radio buttons in this instance, and commented out the old switch rather than removing it completely.

Reply:
ah, I know about `<!--`, but when I read `<!--iv` and then `</div-->` my brain gone _brrr what is this new magic **ie6** compatibility style_ 😆